web/portal/src/main/resources/locale/portal/webui_fr.properties

```diff
@@ -1341,4 +1341,12 @@ onboarding.email.welcome=Vous êtes invité(e) à rejoindre la plateforme
 onboarding.email.accessLink=Pour y accéder, veuillez cliquer sur le lien suivant
 onboarding.email.join=Rejoindre
 onboarding.email.bye=Merci et très belle collaboration au sein de votre communauté!
-onboarding.email.email.footer=eXo Platform
\ No newline at end of file
+onboarding.email.email.footer=eXo Platform
+onboarding.login.create=Créer
+onboarding.login.welcomeTo=Bienvenue sur
+onboarding.login.Hello=Bonjour
+onboarding.login.loginText=Voici votre identifiant :
+onboarding.login.ConfirmPassword=Confirmer votre mot de passe
+onboarding.login.condition=Il faudra se connecter à la plateforme et rejoindre vos communautés.
+onboarding.login.allowCreatePassword=Nous vous permettons de créer votre mot de passe ci-dessous.
+onboarding.login.capatcha=Veuillez cocher la case suivante pour continuer.
```

Comment:
FYI these properties values will be overwritten by Crowdin. It would be better to add these values on Crowdin after PR merge and next Crowdin sync cc @azayati

Reply:
yes yes, we just put them to test on acceptance